### PR TITLE
Add unit testing

### PR DESCRIPTION
**Pull request by omarb147**

### CODE REVIEW

File: jest.config.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
```
Comment (alexwong-crc) [inline jest.config.ts]: I find it frustrating when there are too many config files in the root as they sort of pollute the root dir and makes it hard to locate. What do you think of having a config folder and we put our webpack, babel, jest and tsconfig in it? Don't have to be done in this PR, but what are your thoughts?

File: src/components/Card/tests/Card.test.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+import { render } from '@testing-library/react';
+import Card from '../Card';
+
+describe('<Card/>', () => {
+  it('should render a card with color and text', () => {
+    const size = { width: 100, height: 200 };
+    const { getByText } = render(<Card text="Hello World" color="yellow" size={size} />);
+    expect(getByText('Hello World')).toMatchSnapshot();
+  });
+});
```
Comment (alexwong-crc) [inline src/components/Card/tests/Card.test.tsx]: so far, testing library seems easy enough :D

Why are we only snapshotting the div that contains the text? Should we not snapshot the whole thing?

File: jest.config.ts (hunk 1 above)
Comment (omarb147) [reply jest.config.ts]: Sounds good i'll i implement it on this branch

File: src/components/Card/tests/Card.test.tsx (hunk 2 above)
Comment (omarb147) [reply src/components/Card/tests/Card.test.tsx]: yeah you're right, I should be getting the snapshot of the whole container

**Pull request merged**